app/resources/xslt/Generic.xsl

```diff
@@ -0,0 +1,703 @@
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" version="1.0">
+  <xsl:output method="html"/>
+  <xsl:template match="XMLFragment">
+    <xsl:apply-templates/>
+  </xsl:template>
+  <!--
+  Top level definitions
+  -->
+  <xsl:template match="emph[@altrender='doctype']">
+	</xsl:template>
+  <xsl:template match="emph[@altrender='scope']">
+    <xsl:value-of select="text()"/>
+  </xsl:template>
+  <xsl:template match="scopecontent">
+    <xsl:apply-templates/>
+  </xsl:template>
+  <xsl:template match="bioghist">
+    <xsl:apply-templates/>
+  </xsl:template>
+  <xsl:template match="arrangement">
+    <xsl:apply-templates/>
+  </xsl:template>
+  <xsl:template match="unittitle">
+    <xsl:apply-templates/>
+  </xsl:template>
+  <!--
+  Hyperlink definitions
+  -->
+  <xsl:template match="archref">
+    <xsl:choose>
+      <xsl:when test="@href!=''">
+        <xsl:element name="a">
+          <xsl:attribute name="href">
+            <xsl:choose>
+              <xsl:when test="contains (@href,'&quot;')">
+                <xsl:value-of select="substring-before(substring-after(@href,'&quot;'),'&quot;')"/>
+              </xsl:when>
+              <xsl:otherwise>
+                <xsl:value-of select="@href"/>
+              </xsl:otherwise>
+            </xsl:choose>
+          </xsl:attribute>
+          <xsl:attribute name="title">Opens in a new tab</xsl:attribute>
+          <xsl:if test="not ( contains (@href,'www.nationalarchives.gov.uk') )">
+            <xsl:attribute name="target">_blank</xsl:attribute>
+          </xsl:if>
+          <xsl:apply-templates/>
+        </xsl:element>
+      </xsl:when>
+      <xsl:otherwise>
+        <xsl:apply-templates/>
+      </xsl:otherwise>
+    </xsl:choose>
+  </xsl:template>
+  <xsl:template match="extref">
+    <xsl:choose>
+      <xsl:when test="@href!=''">
+        <xsl:element name="a">
+          <xsl:attribute name="href">
+            <xsl:choose>
+              <xsl:when test="contains (@href,'&quot;')">
+                <xsl:value-of select="substring-before(substring-after(@href,'&quot;'),'&quot;')"/>
+              </xsl:when>
+              <xsl:otherwise>
+                <xsl:value-of select="@href"/>
+              </xsl:otherwise>
+            </xsl:choose>
+          </xsl:attribute>
+          <xsl:attribute name="title">Opens in a new tab</xsl:attribute>
+          <xsl:if test="not ( contains (@href,'www.nationalarchives.gov.uk') )">
+            <xsl:attribute name="target">_blank</xsl:attribute>
+          </xsl:if>
+          <xsl:apply-templates/>
+        </xsl:element>
+      </xsl:when>
+      <xsl:otherwise>
+        <xsl:apply-templates/>
+      </xsl:otherwise>
+    </xsl:choose>
+  </xsl:template>
+  <xsl:template match="ref">
+    <xsl:choose>
+      <xsl:when test="@href!='' and not(@target)">
+        <xsl:element name="a">
+          <xsl:attribute name="href">
+            /catalogue/search/?_q=<xsl:value-of select="."/>&amp;_hb=tna
```

Comment:
Issue: 
http://localhost:65533/catalogue/id/C244/
RG 26 shows link as http://localhost:65533/catalogue/search/?_q=RG%2026&_hb=tna%0A%20%20%20%20%20%20%20%20%20%20
with invalid query params
```suggestion
            /catalogue/search/?q=<xsl:value-of select="."/>
```
The default `group=tna` is internally supplied